test/e2e/specs/test.spec.js

```diff
@@ -1,15 +1,17 @@
+import { NETWORKS } from '../constants';
 import {
   mnemonics,
   accountAddresses,
-  EMERYNET_FAUCET_URL,
   DEFAULT_TIMEOUT,
   DEFAULT_TASK_TIMEOUT,
   DEFAULT_EXEC_TIMEOUT,
-  AGORIC_ADDR_RE,
+  AGORIC_NET,
 } from '../test.utils';
 describe('Wallet App Test Cases', { execTimeout: DEFAULT_EXEC_TIMEOUT }, () => {
   context('Test commands', () => {
     it(`should connect with Agoric Chain`, () => {
+      cy.task('info', `AGORIC_NET: ${AGORIC_NET}`);
```

Comment:
is this needed? or was it for testing?

Reply:
It is not required but I want to retain it to have an idea of the `AGORIC_NET` value